JakinChan200/CS180_Roald
# Adjust client backend_url

## Overview
Quick fix to change the backend url to the /api/ route 

Includes changes from #8 , will merge after

## Materials
N/A

## Pictures
N/A

## Manual Testing Guide
- Go to the home page
- Note when the server is running the message is different

Pull requests should receive approval before merge.
